packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -5,6 +5,7 @@ import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { useTheme } from '../../themes/ThemeContext';
 import { GrafanaTheme } from '@grafana/data';
 import { IconSize, IconName } from '../../types';
+import mdx from './IconButton.mdx';
```

Comment:
You still need to enable the mdx by adding `page: mdx,` inside the docs on line 15.